### PR TITLE
cewl: add page

### PR DESCRIPTION
**Pull request by dhondta**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by sbrl** [comment]
Hrm, looks like Travis has gone weird.

**Comment by zdroid** [comment]
@sbrl I'm here especially to make Travis content. It's my job.

File: pages/linux/cewl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cewl
+
+> URL spidering tool for making a cracking wordlist from web content.
```
Comment (agnivade) [inline pages/linux/cewl.md:3]: I don't understand what is a cracking wordlist. Can you explain ?

Comment (sbrl) [reply pages/linux/cewl.md:3]: /cc @dhondta

Comment (dhondta) [reply pages/linux/cewl.md:3]: @agnivade See https://en.wikipedia.org/wiki/Password_cracking
@sbrl What do you mean with `/cc` ?

Comment (zdroid) [reply pages/linux/cewl.md:3]: /cc like [CC in email](https://en.wikipedia.org/wiki/Carbon_copy#Email). Just a fancy way to mention you. /ping @dhondta is another fancy way.

Comment (agnivade) [reply pages/linux/cewl.md:3]: Thanks for the clarification. I initially thought it was too difficult for a beginner to understand, but now looking at it again, I think it's easy enough.

**Pull request merged**